src/packages/pvm/pvm/disassemblify.ts

```diff
@@ -20,7 +21,10 @@ export function disassemblify(rawProgram: Uint8Array) {
 
   while (i < code.length) {
     const currentInstruction = code[i];
-    const argumentType = instructionArgumentTypeMap[currentInstruction];
+    const isValidInstruction = currentInstruction < instructionArgumentTypeMap.length;
```

Comment:
This condition works because currently all values from `0` to `instructionArgumentTypeMap.length - 1` are valid instructions but it this not guaranteed anywhere so it would be safer to do something like:

```suggestion
    const isValidInstruction = Instruction[currentInstruction] !== undefined;
```